src/locales/en/product.json

```diff
@@ -55,7 +55,10 @@
         "loadMore": "Load more...",
         "hideBtn": "Hide comments"
     },
-    "images": {
-        
+    "snackBar": {
+        "added": "Your comment is successfully added",
+        "addedReply": "Your answer is successfully added",
+        "deleted":  "Your comment is successfully deleted",
+        "error":  "Something went wrong"
```

Comment:
Maybe it will be better to move the error message to locales - errors, because it is used on different pages

Reply:
yes , it's better idea, have fixed

Reply:
nice) also please delete this unneeded line 62 in locales en.